remark/remark.ts

```diff
@@ -54,15 +54,15 @@ export class MarkdownEngine implements Engine {
   ): Promise<string> {
     return (await this.engine.process({
       value: content,
-      data: data || {},
+      data: data.page.data || {},
```

Comment:
I'd include a `?` to ensure the script doesn't fail if the `page` doesn't exist.

```suggestion
      data: data.page?.data || {},
```

Reply:
Done